PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt

```diff
@@ -82,10 +79,6 @@ class PayPalVaultRequest
             .put(CANCEL_URL_KEY, cancelUrl)
             .put(OFFER_CREDIT_KEY, shouldOfferCredit)
 
-        shippingCallbackUrl?.let {
-            if (it.toString().isNotEmpty()) parameters.put(SHIPPING_CALLBACK_URL_KEY, it)
-        }
```

Comment:
Should this logic move into `PayPalCheckoutRequest`?

Reply:
Logic is already there https://github.com/braintree/braintree_android/blob/388c2d64336e0a94ad83995f9ac0c7ffc4e49d32/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt#L113 thanks for asking

My mistake was approaching this change differently in iOS. In iOS, I created the feature branch from scratch, while here, I only removed `shippingCallbackUrl` from the Vault flow, keeping the changes from the PR previously merged into this feature branch.